spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -158,6 +158,10 @@ def assert_specific_vm
       )
 
       expect(vm.snapshots.first.total_size).to be > 0
+
+      instance_name = "test-instance-rhel-s922-shared-tier3"
```

Comment:
Could we put `test-instance-ibmi-s922-capped-tier1` in a resource_pool so that we don't have to check a second vm?

Reply:
I tried once, but if `placement_group` and `spp_placement_group` contradict each other, trying to put the VM on this or that host, then it'd fail, and it did fail on me trying to provision a VM with both specified. I figured that'd be just asking for a trouble at a later time, kinda relying on a luck.